addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -350,6 +350,7 @@ export class WysiwygAdapterComponent extends ComponentAdapter {
     _addEditorMessages() {
         const $wrap = this.$editable.find('.oe_structure.oe_empty, [data-oe-type="html"]');
         this.$editorMessageElement = $wrap.not('[data-editor-message]')
+                .attr('data-editor-message-default', true)
```

Comment:
Meh, don't like giving a non-string to `attr` but I guess this will go away in a no-jquery refactoring at some point anyway :+1: